test/addons-napi/test_promise/test.js

```diff
@@ -1,17 +1,18 @@
 'use strict';
 
 const common = require('../../common');
-const test_promise = require(`./build/${common.buildType}/test_promise`);
 const assert = require('assert');
 
+// Testing api calls for promises
```

Comment:
Can you format the comment as what [testing guide](https://github.com/nodejs/node/blob/master/doc/guides/writing-tests.md#test-structure) suggests?

```
'use strict';

const common = require('../../common');

// This tests the promise-related n-api calls

const assert = require('assert');
const test_promise = require(`./build/${common.buildType}/test_promise`);
```

Reply:
done!